themes/beaver/assets/css/technologies.css

```diff
@@ -0,0 +1,70 @@
+.technologies__container--white, .technologies__container--white h2 {
+    color: #fff!important;
+}
+
+.technologies__subheading {
+    color: #1a8cff;
+    font-weight: 600;
+}
+
+.technologies--header {
+    text-align: center;
+    padding-top: 130px;
+    padding-bottom: 60px;
+}
+
+.technologies--header h2 {
+    margin-bottom: 20px;
+    margin-top: 30px;
+}
+.technologies__container {
+    font-size: 0;
+    text-align: center;
+}
+
+
+.technologies__item {
+    font-size:18px;
+    display: inline-block;
+    padding: 20px 0;
+    box-sizing: border-box;
+    width: 25%;
+    font-weight: bold;
+}
+
+.technologies__container img, .technologies__container svg {
+    width: 65px;
+    height: auto;
+    margin-bottom: 20px;
+}
+
+@media (max-width: 850px) {
+    .technologies__item {
+        width: 50%;
+    }
+}
+
+@media (min-width: 980px) {
+    .technologies--header {
+       margin-left: 100px; margin-right: 100px;
+    }
+    .technologies__description {
+        margin-left: 200px;
+        margin-right: 200px;
+    }
+}
+
+
+/*
+quick fix services page
+ */
+
+.fl-node-w8dym2q746pj>.fl-row-content-wrap {
+    padding-top: 0;
+}
+/*
+quick fix use-case page
+ */
+.fl-node-r5xy9wufkc2e>.fl-row-content-wrap {
+    padding-top: 0;
+}
```

Comment:
We agreed to use  rscss , please update it

Reply:
ok